### PR TITLE
refactor: semantic markup improvement, fix validation warnings

### PR DESCRIPTION
**Pull request by Josh-Cena**
## Motivation

Did some HTML validation. The most alerting issue is "`<main>` cannot be nested" and "one page should only have one `<main>`".

In addition, I've marked one text as translatable, and removed the deprecated `frameBorder` attribute.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests)?

Yes

### CODE REVIEW

File: packages/docusaurus-theme-classic/src/theme/DocCardList/index.tsx [hunk 1]
```diff
@@ -27,9 +27,9 @@ export default function DocCardList({items, className}: Props): JSX.Element {
   return (
     <div className={clsx('row', className)}>
       {filterItems(items).map((item, index) => (
-        <article key={index} className="col col--6 margin-bottom--lg">
```
Comment (lex111) [inline packages/docusaurus-theme-classic/src/theme/DocCardList/index.tsx:30]: Why change the `article` with `section` here? Can't we keep the `article` as repeatable item? I would change the parent `div` with `row` class to `section` tag in that case.

Comment (Josh-Cena) [reply packages/docusaurus-theme-classic/src/theme/DocCardList/index.tsx:30]: So, articles within sections? I thought sections should be descendants of articles, though. I'm fine with making the card and the list both `article`.

Comment (Josh-Cena) [reply packages/docusaurus-theme-classic/src/theme/DocCardList/index.tsx:30]: Ah, sorry, misread. Makes sense.

**Review by lex111 — APPROVED** [review]
:+1:

**Pull request merged**